beacon_node/execution_layer/src/engine_api/http.rs

```diff
@@ -72,12 +78,15 @@ pub static LIGHTHOUSE_CAPABILITIES: &[&str] = &[
 pub static PRE_CAPELLA_ENGINE_CAPABILITIES: EngineCapabilities = EngineCapabilities {
     new_payload_v1: true,
     new_payload_v2: false,
+    new_payload_v3: false,
     forkchoice_updated_v1: true,
     forkchoice_updated_v2: false,
+    forkchoice_updated_v3: false,
     get_payload_bodies_by_hash_v1: false,
     get_payload_bodies_by_range_v1: false,
     get_payload_v1: true,
     get_payload_v2: false,
+    get_payload_v3: false,
```

Comment:
Looks like we should be able to remove this now as `ENGINE_EXCHANGE_CAPABILITIES` endpoint is now expected to exist?

Reply:
I think we can just straight up delete this whole constant

Reply:
Issue: #4848